com.unity.formats.fbx/Tests/FbxTests/ModelExporterTest.cs

```diff
@@ -1227,5 +1228,72 @@ public void TestMissingMeshExport()
             ModelExporter.ExportObject(filename, cube);
             Assert.IsNotNull(filename);
         }
+
+        [Test]
+        public void TestMaterialScaleAndOffset() 
+        {
+            string matClampGuid = "e6598d8bd228e5940988877e9eddd594";
+            string matRepeatGuid = "f07bd71d1d87a7b41acfc483bf06be3f";
```

Comment:
this var doesn't seem to be used, I think there is a typo on line 1245 which uses the clamp guid instead of this one

Reply:
Fixed, good catch